\documentclass[journal=jctcce,manuscript=article]{achemso}

\usepackage[version=3]{mhchem}
\usepackage[T1]{fontenc}       
\usepackage{graphicx}
\usepackage[table,xcdraw]{xcolor}
\usepackage{subcaption}

\author{Nathan M. Lim}
\email{dmobley@mobleylab.org}
\affiliation[University of California---Irvine]
{Department of Pharmaceutical Sciences, University of California---Irvine, Irvine, California 92697, United States}
\author{Lingle Wang}
\affiliation[Schr\"{o}dinger, Inc.]
{Schr\"{o}dinger, Inc., 120 West 45th Street, New York, New York 10036, United States}
\author{Robert Abel}
\affiliation[Schr\"{o}dinger, Inc.]
{Schr\"{o}dinger, Inc., 120 West 45th Street, New York, New York 10036, United States}
\author{David L. Mobley}
\affiliation[University of California---Irvine]
{Department of Pharmaceutical Sciences, University of California---Irvine, Irvine, California 92697, United States}

\title{Sensitivity in binding free energies due to protein reorganization}

\begin{document}

\begin{abstract}
Tremendous recent improvements in computer hardware, coupled with advances in sampling techniques and force fields, are now allowing protein-ligand binding free energy calculations to be routinely used to aid pharmaceutical drug discovery projects. 
However, despite these recent innovations, there are still needs for further improvement in sampling algorithms to more adequately sample protein motion relevant to protein-ligand binding. 
Here, we report our work identifying and studying such clear and remaining needs in the apolar cavity of T4 Lysozyme L99A. 
In this study, we model recent experimental results that show the progressive opening of the binding pocket in response to a series of homologous ligands \cite{Merski2015}.
Even while using enhanced sampling techniques, we demonstrate that the predicted relative binding free energies (RBFE) are sensitive to the initial protein conformational state.
Particularly, we highlight the importance of sufficient sampling of protein conformational changes and demonstrate how inclusion of three key protein residues in the 'hot' region of the FEP/REST simulation improves the sampling and resolves this sensitivity.
\end{abstract}

\begin{tocentry}
\includegraphics[width=9cm,height=3.5cm]{Figures/T4-L99A_cover.pdf}
\end{tocentry}

\section{Introduction}
Proteins play a central role in many biological processes by modulating many key signaling pathways.
It is unsurprising to find that proteins make up the vast majority of pharmaceutical drug targets.
Many small-molecules on the market today induce their therapeutic effect by modulating the proteins' biological activity through binding\cite{overington2006many,FCP:FCP548,Lundstrom2009}. 
Thus, optimization of protein-ligand binding affinities is a central goal in early pharmaceutical drug design projects.

Recent advancements in technology and computational chemistry have led to increasing use of computer-aided drug design techniques to assist in lead optimization in pharmaceutical drug discovery.
Since accuracy and reliability of the approach being used is critical for success, it is here where the most rigorous of methods like free energy calculations can be applied in the prediction of protein-ligand binding affinities. 
Using alchemical methods like free energy perturbation (FEP), thermodynamic integration, and $\lambda$-dynamics on molecular dynamics (MD) simulations, the difference in binding free energies between two ligands can be computed in a robust and accurate manner.
By computing relative binding free energies, much of the computational cost and difficulties of absolute binding free energy calculations is avoided\cite{doi:10.1021/ct5000296,chipot2007free,chodera2011alchemical,knight2009lambda,zheng2008random,gallicchio2011advances,doi:10.1021/ct500161f}.
Other advancements in forcefields, sampling algorithms, and emergence of GPU computing have considerably improved the accuracy and robustness of alchemical calculations. 
The recent development of FEP+, a fully automated alchemical protocol implemented with modern methodologies, reduces overall workload and the potential for human error in setup and analysis of these types of calculations.
Using this protocol, previous studies\cite{FEPplus} have demonstrated it to yield highly accurate free energy predictions in a wide range of pharmaceutically relevant protein targets and ligands.
The accuracy and reliability of FEP+ makes it a very powerful tool in the hands of medicinal chemists for efficient optimization of lead compounds.

Although the previous report on FEP+\cite{FEPplus} demonstrates its robustness, the primary concern focuses on the size of the ligand perturbation or the initial ligand pose and how these factors impact the accuracy of calculated free energies\cite{mobley2012perspective,doi:10.1021/acs.jctc.5b00214}.
Careful consideration of effects arising from protein conformational changes have generally received less attention due to the difficulty in addressing the sampling challenges encountered when simulating proteins.
The difficulty in protein sampling lies primiarly in the timescales required to adequately capture a complete biomolecular event. 
Timescales of such events can range from nanoseconds to milliseconds (or even seconds)\cite{elber2005long}.
In MD simulations, the system evolves in time through series of short time steps (i.e. 2 fs) by repeatedely computing the forces on each individual atom in accordance to Newton's laws of motion at each time step.
Not only must a computer perform numerous calculations at each time step, but it must repeat this process an enourmous amount of times to generate a trajectory that is on timescale for your biological process of interest\cite{karplus2005molecular}.
Furthering the difficulty, with using `vanilla' MD simulations, the system will more than likely remain kinetically trapped in an energy minima, thereby preventing any further exploration of the protein conformational space and obtaining proper sampling.
Some computational studies have shown even small simple changes in the protein---like a side-chain rotamer flipping---can cause large errors in affinity predictions if not properly sampled\cite{Mobley2009489,Mobley20071118,Jiang:2010tg,Meng:2015gj}.
Overcoming these high energy barriers is no easy feat; whereby a number of distinct approaches like metadynamics\cite{laio2002escaping}, accelerated MD\cite{hamelberg2004accelerated}, and temperature accelerated MD\cite{Maragliano2006168} aim to solve this problem of kinetic traps but may fall short in some cases\cite{borhani2012future}.
Thus, careful sampling of protein conformational changes is critical for accurate and reliable affinity predictions. 

In the previous FEP+ study\cite{FEPplus}, other than simply restricting ourselves to congeneric series with multiple reported crystal structures, no effort was made to either include or exclude cases where the protein or ligand binding mode may reorganize.  
However, it is entirely possible some of the outliers reported in that work may be due to protein conformational reorganization.
The testing of the FEP+ technology on cases where protein reorganization has been well-characterized experimentally may provide an opportunity to probe potential pathological cases, and further provides an opportunity to discover how to further improve the technology. While some studies suggest protein reorganization on ligand binding can be relatively common (i.e. as reviewed by~\cite{Mobley:2009bm}), it is far from well understood how often this can be a challenge for binding prediction.  
Regardless, it is clear that protein-ligand binding can be highly complex, and multiple stable protein conformations may be populated even for the same ligand \cite{doi:10.1021/jm060167o,Gutteridge200521,Merski2015}.
Thus, there is a great need to understand how protein conformational changes and the choice of starting protein conformation may affect the reliability of free energy predictions.  

Here, we apply the now-standard FEP protocol used in ~\cite{FEPplus}, which utilizes replica exchange with solute tempering (FEP/REST)\cite{FEP/REST}, to a very simple model binding site in an engineered mutant of T4 lysozyme (L99A).
In this mutant, the L99A mutation creates a small apolar binding site that has been studied extensively experimentally\cite{eriksson1992response,eriksson1993similar,T4affinity,doi:10.1021/bi00027a007} and computationally by docking\cite{wei2002model,wei2004testing,graves2005decoys,Merski2015} and free energy methods\cite{Mobley20071118,hermans1997inclusion,boresch2003absolute,deng2006calculation,mann2000modeling,Boyce2009,FEP/REST,FEP/RESTapp}.
Recent studies on this binding site have found that the protein adopts three discrete conformations in response to ligand binding\cite{Merski2015}.
Through a series of eight congeneric ligands (Fig~\ref{fig:ligand_set}), each growing by addition of a single methyl, the protein responds by a single helix rearrangement to accommodate the growing ligand (Fig~\ref{fig:T4-L99A}).
As the ligand grows, the binding cavity was observed to incrementally reorganize into three discrete conformations which we will refer to as the closed (Fig~\ref{fig:closed_surface}), intermediate(Fig~\ref{fig:int_surface}), and open states (Fig~\ref{fig:open_surface}).
Using the FEP/REST protocol and the aforementioned homologous ligand series, we calculate relative protein-ligand binding affinities between ligands that occupy different discrete protein conformational states.
In this study---while using the implemented default FEP/REST protocol---we demonstrate how the kinetically distinct protein states and structural rearrangement affects the accuracy and reliability of our predicted relative binding affinities.
Further, we illustrate the importance of sufficient sampling of protein conformational changes and how modification of the REST region can potentially address this issue.

\section{Methods}
\subsection*{Protein/Ligand preparation}
All proteins were prepared and aligned in Maestro\cite{Maestro} using the `Protein Preparation Wizard'\cite{ProteinPrepWizSoftware,Epik,Impact,Prime,ProteinPrepWizPaper} tool and with the following settings enabled (as they appear in the Maestro GUI menu):
   \begin{itemize}
   \item Preprocess: Assign bond orders, Add hydrogens, Create zero-order bonds to metals, Create disulfide bonds, Cap termini, Delete waters beyond 5\AA{} from het groups
   \item Refine: Sample water orientations, Use PROPKA pH: 7.0, Remove waters with less than 3 H-bonds to non-waters, and restrained minimization.
   \end{itemize}

Protein structures were taken from PDBs: 4W52, 4W53, 4W54, 4W55, 4W56, 4W57, 4W58, and 4W59 corresponding to protein-ligand bound structures of benzene, toluene, ethylbenzene, propylbenzene, sec-butylbenzene, butylbenzene, pentylbenzene, and hexylbenzene, respectively\cite{Merski2015}.
Each simulation starts from either the protein closed state (PDB:4W52) or the open state (PDF:4W59).
Using LigandFEP, our system preparation follows a similar workflow to the tutorial\cite{LigandFEP}.
Generally, two options were taken:\\
(1a) If the simulation starts from the protein closed state, the benzene crystal position was used as a reference for fragment building (PDB:4W52).\\
(1b) The corresponding ligand in the transformation was built by duplicating benzene in place and adding methyl groups.\\
(2a) If the simulation starts from the protein open state, the hexylbenzene crystal position was used as reference for fragment building (PDF:4W59).\\
(2b) The corresponding ligand in the transformation was built by duplicating hexylbenzene in place and deleting methyl groups.\\
Ligand tail fragments were added using the Build/Fragments toolbar in Maestro  and were not overlaid or docked. 
As the ligand tails were built, bonds were manually rotated so that the tail was oriented in a similar manner as in their corresponding crystal structure.
Following, the newly added atoms in the tail were locally minimized while leaving the core in its initial position.
This was done in an attempt to correct bond angles and minimize the core RMSD, which LigandFEP uses to determine the core atoms between the two ligands.

\subsection*{Classification of alchemical transformations and color coding}
Here, we classified ligands based on the primary protein conformation (closed, intermediate, or open) the ligand occupies from the experimental studies (Table~\ref{tbl:expdata}).
To be explicit, the set of closed ligands refers to benzene, toluene, ethylbenzene, and propylbenzene; (sec-)butylbenzene for intermediate; and pentyl/hexylbezene for open ligands.
The various protein states and ligands are then assigned a color accordingly: purple for closed, cyan for intermediate, and green for the open state.

\begin{table}[!htb]
\centering
\caption{Experimental protein-ligand occupancies and affinities\textsuperscript{\emph{a}}}
\label{tbl:expdata}
\begin{tabular}{|c|c|c|c|c|c|c|c|}
\hline
\textbf{PDB}  & \textbf{Ligand} & \textbf{C} & \textbf{I} & \textbf{O} & \boldmath$\Delta G_{exp}$  & \boldmath$\sigma_{exp}$ &  \textbf{Set} \\ \hline
4W52   &  benzene          & \cellcolor[HTML]{C0C0C0}90   & -     & -    & -5.19      & 0.16       &  C             \\ \hline
4W53   &  toluene          & \cellcolor[HTML]{C0C0C0}80   & 20   & -    & -5.52      & 0.04       & C   \\ \hline
4W54   &  ethylbenzene     & \cellcolor[HTML]{C0C0C0}50    & 50   & -    & -5.76      & 0.07       & C   \\ \hline
4W55   &  propylbenzene  & \cellcolor[HTML]{C0C0C0}60    & 40   & -    & -6.55      & 0.02       & C   \\ \hline
4W56   &  sec-butylbenzene & 40        & \cellcolor[HTML]{C0C0C0}60      & -    & N/A      & -     &  I   \\ \hline
4W57   &  butylbenzene   & 10        & \cellcolor[HTML]{C0C0C0}60      & 30   & -6.70   & 0.02  &  I   \\ \hline
4W58   &  pentylbenzene  & 30        &  -       & \cellcolor[HTML]{C0C0C0}70  & N/A     & -      &  O   \\ \hline
4W59   &  hexylbenzene   & 30        &  -       & \cellcolor[HTML]{C0C0C0}70  & N/A     & -     & O   \\ \hline
\end{tabular}

\textsuperscript{\emph{a}} Listed are the PDB codes of the corresponding protein-ligand complex crystal structures and the ligand occupancies\cite{Merski2015} for the closed(C), intermediate(I), and open(O) protein conformations by percentage. 
Under the `Set' column, shaded in grey, is each ligand's primary state of occupancy which we use to categorize our alchemical transformation sets.
Binding affinities for pentylbenzene and hexylbenzene were inaccessible due to solubility limits\cite{Merski2015}. 
Experimental binding free energies and their uncertainties are in units of kcal/mol\cite{T4affinity}.
\end{table}

Our sets of alchemical transformations consisted of 3 groups: `closed-intermediate', `closed-open', and an `experimental' ligand set.
The first two sets were classified based on the expected conformational change that would result from the alchemical transformation.
For example, the alchemical transformation of benzene to butylbenzene falls into closed-intermediate while benzene to hexylbenzene is a closed-open transformation.
In our experimental set, we perform all possible combinations of transformations for ligands with available experimental binding affinities (Table~\ref{tbl:expdata}).
Ligands with available experimental binding affinities consisted of benzene, toluene, ethylbenzene, propylbenzene, and butylbenzene.
This gives a total of 26 alchemical transformations in this study, 8 from `closed-intermediate', 8 from `closed-open', and 10 from the experimental set.
 
\subsection*{FEP protocols}
Using the Schr\"{o}dinger application suite (release 2015-3)\cite{Maestro-Desmond}, we utilize a similar protocol to FEP+\cite{FEPplus} called LigandFEP\cite{LigandFEP}.
FEP+ is a fully automated work flow that plans perturbation pathways based a variant of LOMAP\cite{LOMAP} mapping algorithm which uses the maximum common substructure (MCS) between any pair of compounds.
LigandFEP is an academic toolkit that generates the configuration files to perform the free energy calculation but is limited in the sense that the user must plan each perturbation path instead.
Both LigandFEP and FEP+ use the same Desmond relaxation protocol and the FEP/REST methodology\cite{REST,REST2,FEP/REST,FEP/RESTapp}. 
By utilizing LigandFEP, we demonstrate LigandFEP can be a powerful tool for academics as it strictly differs only in the level of automation.

\subsection*{REST region selection}
In this study, by default, only heavy atoms in the ligand were included in the REST region unless specified otherwise.
Further details on the temperature profile and how the REST region is normally selected can be found in previous studies\cite{FEP/REST,FEPplus} in the supporting information.
Simulations that included protein heavy atoms in the REST region are referred to with the `pREST' label, where selection of the particular residues is described as follows.

Based on visual inspection of our molecular dynamics simulations and considering the F-helix spans residues 107-115, we selected residues Glu108, Val111, and Gly113 to include into the REST region (Fig~\ref{fig:C2O}).
Glu108 sits near the start of the helix which appears as a hinge point for the opening and closing of the binding cavity (Fig~\ref{fig:Glu108-C2O}).
Following, Val111 appears in the middle of the helix and was observed to undergo the largest motion during protein conformational changes (Fig~\ref{fig:Val111-C2O}).
Gly113 was included in order to collectively have hot regions approximately at the start, middle and end points of the helix.

\subsection*{Simulation details}
Desmond\cite{DESMONDSoftware,DESMONDarticle,DESMONDPaper1,DESMONDPaper2} simulation protocols have been described previously in the supporting information\cite{FEPplus} or can be found in greater detail in the Desmond User Manual\cite{DESMONDManual}.
The relaxation protocol begins with a simulation where solute molecules are restrained to their initial positions while minimizing using a Brownian dynamics NVT integrator for 100ps, followed by 12ps simulations at 10K with a NVT ensemble and then a NPT ensemble using the Langevin method\cite{Langevin}.
Next is a 24ps simulation followed by a final 240ps simulation with solute molecules unrestrained, both are carried at room temperature with a NPT ensemble using Langevin.
Production simulations with the default REST region were ran for the default setting of 5ns. 
Initial `pREST' production simulations were also simulated for 5ns and then were carried out to a length of up to 55ns for all closed-open transformations. 
For closed-intermediate transformations, we extended `pREST' production simulations up to 25ns, only for cases that were far from convergence with the 5ns simulation time.
Here, we use the final 15ns for closed-open simulations and the final 10ns for closed-intermediate to calculate our final free energies, discarding the initial time as additional equilibration time.
FEP/REST simulations were run on four GeForce GTX Titan Black GPUs using the Desmond/GPU engine with the recently developed OPLS3\cite{OPLS3} forcefield parameters.

\subsection*{Calculation of free energies and measurement of inconsistency}
Throughout this study, we measure the inconsistency ($\Delta\Delta G_{\boldsymbol{\varepsilon_n}}$) between the final calculated free energies between simulations that start from the protein closed state ($\Delta\Delta G_{\mathbf{C_n}}$) versus the protein open state ($\Delta\Delta G_{\mathbf{O_n}}$) by simply taking the difference.
\begin{equation}
  \mathbf{\Delta\Delta G_{\boldsymbol{\varepsilon_n}}}  = \left |  \Delta\Delta G_{\mathbf{C_n}  } - \Delta\Delta G_{\mathbf{O_n} }\right | 
  \label{eqn:diffG}
\end{equation}
Then we compute the overall inconsistency---referred to as the 'Root-Mean-Square-Inconsistency' (RMSI)---for each set of alchemical transformations.
The RMSI is calculated by using the differences ($\Delta\Delta G_{\boldsymbol{\varepsilon_n}}$) obtained from the comparisons between protein open and closed simulations.
\begin{equation}
\mathbf{RMSI} = \sqrt{   \frac{ \sum_{n} (\Delta\Delta G\varepsilon_{n} )^2  } {n}}
  \label{eqn:RMSI}
\end{equation}
Similarly, we compute the 'Root-Mean-Square-Error'(RMSE) when comparing with experimental free energies for both simulations starting from the protein closed and open state.
\begin{equation}
\mathbf{RMSE^{O}} = \sqrt{   \frac{ \sum_{n} (\Delta\Delta G_{\mathbf{O_n}} - \Delta\Delta G_{\mathbf{exp_n}} )^2  } {n}}
\mathbf{RMSE^{C}} = \sqrt{   \frac{ \sum_{n} (\Delta\Delta G_{\mathbf{C_n}} - \Delta\Delta G_{\mathbf{exp_n}} )^2  } {n}}
  \label{eqn:RMSE}
\end{equation}
Calculated free energies were determined using the Bennett acceptance ratio\cite{BAR} (BAR) with error estimations using both bootstrapping and BAR analytical error prediction\cite{BARerror}.
Hysteresis around closed thermodynamic cycles and best estimates of the free energies with their errors were calculated using the cycle closure algorithm discussed in a previous publication\cite{FEP/REST}.

\subsection*{Determining the protein conformation state using RMSD}
In this study, we determine the state of the protein by computing the 'Root-Mean-Square-Deviation' (RMSD) of the protein backbone atoms spanning the F-helix relative to their positions found in the closed (PDB:4W53), intermediate (PDB:4W57), and open (PDB:4W59) crystal structures (Fig~\ref{fig:T4-L99A_tube}).
The set of RMSDs---that is, the RMSD relative to the closed, intermediate, and open states---is computed at each frame over the course of the entire simulation. 
Then, we use the protein conformational state with the lowest RMSD to correspondingly color each time point in our analyses of `RMSD/time' and `Color maps'.
Again, we use purple to denote the protein closed state, cyan for the intermediate state, and green for the open state.
Here, we use VMD \cite{VMDpaper,VMDalignment} to align and compute the RMSD of our Desmond trajectories relative to crystal structures.
Further details on the procedure and the scripts used for these analyses are provided in the supplementary info.

For the `RMSD/time' analysis, see Figure~\ref{fig:o_opls3_1/RMSD-replica11}) for reference. 
Here, we plot the RMSD to the closed helix, represented by the black line, where each time point is colored according to the lowest RMSD state.
We apply the RMSD/time analysis only to the simulation corresponding to the end state ligand of interest ($\lambda_{11}$).
By tracking the RMSD relative to the closed helix, we can monitor if the protein opens (by high RMSD with green points) or closes (by low RMSD with purple points).
Additionally, we gain some insight on the time required to capture the opening or closing of the binding cavity.

It is important to note that by restraining our analysis to only the end-state replica, we limit our ability to completely view the effects of coordinate swapping during replica exchanges.
We address this limitation by analyzing all replicas in what we call `Color maps', see Figure~\ref{fig:c_opls3_rest1_1/colormap} for an example.
Essentially, our 'Color map' analysis is the same as our 'RMSD/time' plots but without the RMSD line plot.
In other words, we color time points according to the protein state of lowest RMSD and do this for all replicas but do not track the RMSD relative to the closed state.
Through a collective view of all replicas, we gain a better perspective of the overall protein conformational sampling and the states they occupy over each replica's separate trajectories. 
Using color maps, it becomes visually easy to see if intermediate---higher temperature---lambda windows are able to sample, say the open state, and if this leads to an enhancement in sampling at the end states via replica exchange.

\section{Results}
\subsection*{Calculated free energies depend strongly on starting protein conformation}
Using the default FEP/REST methodology\cite{FEP/REST}, we find calculated free energies significantly depend on the protein starting conformation, especially for large perturbations (i.e. opening the cavity from the closed state).
To illustrate this, we begin our molecular dynamics simulations both from the protein closed and open conformations then perform alchemical transformations to ligands that occupy another protein conformational state.
For example, in the alchemical transformation of benzene to hexylbenzene---starting from the protein closed state---we expect to see opening of the binding cavity when the ligand is in the fully interacting hexylbenzene state.
In this study, we demonstrate using the default FEP protocol settings of a 5ns simulation time and REST region selection does not generate adequate sampling of the motion in the F-helix and does not eliminate the dependence on the initial protein state.

\subsubsection*{Closed-Open Ligand Transformations}
An examination of the largest alchemical transformation, benzene to hexylbenzene, clearly highlights the sampling challenges faced when using the default FEP/REST protocol.
From experimental data of ligand occupancies (Table~\ref{tbl:expdata}), we expect in our simulations of hexylbenzene to see the protein primarily in the open state over the closed state.
Instead, we find the protein remains trapped in its initial conformational state whether we start from closed (Fig~\ref{fig:c_opls3_1/RMSD-replica11}) or open (Fig~\ref{fig:o_opls3_1/RMSD-replica11}) over the course of the 5ns simulation.
From the protein closed simulation, the protein only briefly samples the intermediate state around 3ns but never enters the open conformation.
As the protein tries to accommodate hexylbenzene and enter its preferred open state, protein-ligand strain results, yielding a positive value for $\Delta\Delta G_{calc}$(+4.13 kcal/mol).
On the other hand, in the protein open simulations, the protein already begins in its preferred state for hexylbenzene and stays only in this open state.
As expected, the $\Delta\Delta G_{calc}$ is negative(-0.61 kcal/mol) as there is no occurrence of large protein-ligand strain in order to open the cavity.
By remaining trapped in the initial state, we under-sample the open state if we begin from the closed state or over-sample it if we begin from the open state.
Ultimately, we arrive at two very different relative free energies values, where the inconsistency is as large as +4.74 kcal/mol for the same transformation of benzene to hexylbenzene.

In the overall set, we similarly observe protein closed simulations to yield positive free energies and negative for protein open simulations. 
In turn, we find the overall inconsistency to be very high with a RMSI of +4 kcal/mol (Fig~\ref{fig:C2O_xyplot}, Table~S\ref{tbl:C-O}). 
Clearly, despite the use of implemented default FEP/REST protocol, we are unable to get sufficient sampling in the protein within the standard 5ns time frame.
This is perhaps unsurprising since in the default calculation, only the perturbed ligand R-group is added to the enhanced sampling region.  
Instead, we encounter sampling problems as the protein remains in its initial conformational state throughout the simulation. 
As a result, our calculated free energies exhibit high dependence on the initial protein configuration which is reflected by the large RMSI.

\subsubsection*{Closed-Intermediate Ligand Transformations}
In the case of closed-intermediate alchemical transformations, we find that the calculated free energies still have some (albeit much smaller) dependence on the initial protein conformation, using the default protocol.
For this set of alchemical transformations, we find the RMSI to be +0.60 kcal/mol (Fig~\ref{fig:C2I_xyplot}, Table~S\ref{tbl:C-I}).
Considering this set involves a smaller protein conformational change and smaller perturbations to the ligand, it is unsurprising to find the RMSI to be much smaller than our closed-open transformation set.

Although, the collective RMSI for closed-intermediate transformations falls in the acceptable range of less than 1 kcal/mol, we can still see a dependence on the initial protein configuration by viewing transformations involving butylbenzene.
For these cases in particular, we observe the same pattern of protein closed simulations yielding positive free energies and negative for protein open simulations. 
We do not see this pattern for transformations with sec-butylbenzene as it does not partially occupy the open state, unlike butylbenzene (Table~\ref{tbl:expdata}).
Through this observation, we demonstrate further that the default protocol does not completely eliminate the free energy dependence on the protein starting conformation, even for smaller perturbations. 
Although, the level of discrepancy (0.6 kcal/mol) is quite small for this set.

\subsubsection*{Experimental Ligand Transformations}
Now, when we compare $\Delta\Delta G_{calc}$ against $\Delta\Delta G_{exp}$, we find that simulations starting from the protein closed conformation are further from converging to $\Delta\Delta G_{exp}$ than when starting from the protein open conformation. 
Here, we calculate the RMS-'Error' with experiment and find the RMSE for protein closed simulations to be +1.0 kcal/mol and +0.58 kcal/mol with protein open (Fig~\ref{fig:exp_xyplot},Table~S\ref{tbl:exp_set}).
Our total RMSI falls within our acceptable range at 0.68 kcal/mol.
However, the fact that protein open simulations are much closer to $\Delta\Delta G_{exp}$, once again does demonstrate our calculated free energies depend on the initial protein state.
Unsurprisingly, the relatively larger RMSE seen for protein closed simulations primarily comes from transformations involving butylbenzene.
Evidently, we find the simulations involving butylbenzene remain trapped in their respective starting conformations, resulting in inadequate sampling in the protein closed simulations (Fig.~\ref{fig:c_exp_opls3_11/RMSD-replica11}) versus the protein open simulations (Fig.~\ref{fig:o_exp_opls3_24/RMSD-replica11}).
Despite performing much smaller alchemical transformations, this shows we still encounter some sampling problems that result in $\Delta\Delta G_{calc}$ that depend on the initial protein conformation, evident when comparing to $\Delta\Delta G_{exp}$.

\subsection*{Including protein residues into the REST region (pREST) improves sampling}
Primarily, we encounter major sampling problems when we begin our simulations from the protein closed state and attempt a mutation which should result in the binding cavity opening.
In order to facilitate protein motion, thereby enhancing protein sampling, we included 3 key residues spanning the F-helix region into the REST region, which we will denote simulations using this with `pREST' (Fig~\ref{fig:C2O}).
By expanding the REST region, we are able to drive the F-helix out its initial state trap by locally heating up key regions and thereby reduce our sampling problem.

To demonstrate the REST improvement over the default protocol, we return to the case of benzene to hexylbenzene.
Here, we show the facilitation of the helix motion by first referring to Figure~\ref{fig:c_opls3_1/RMSD-replica11} which shows that there is no sampling of the open state for the default protocol.
Now with pREST, we see a few open state points around 3ns and even a single open point before closing again after our initial step (Fig~\ref{fig:c_opls3_rest1_1/RMSD-replica11}).
Alternatively, we can further illustrate the enhancement of protein sampling by viewing all replicas collectively, using the color maps.
In reference to Figure~\ref{fig:c_opls3_1/colormap} and Figure~\ref{fig:c_opls3_rest1_1/colormap}, we illustrate that there is far less sampling of the intermediate or open protein states in default simulations versus the pREST simulations.
 
Collectively, we find only some minor improvements in the RMSI for all our closed-open and closed-intermediate transformations while using pREST. 
For closed-open transformations (Table~S\ref{tbl:C-O_pREST}), the RMSI reduces to +2.82 kcal/mol (previously +4 kcal/mol). 
On the other hand, for closed-intermediate, the RMSI raises slightly to +0.79 kcal/mol (previously +0.60 kcal/mol), which may be due to statistical noise (Table~S\ref{tbl:C-I_pREST}).
Generally, simulations starting from the closed state had $\Delta\Delta G_{calc}$ values that moved towards favorability (i.e. more negative $\Delta\Delta G_{calc}$), while for protein open simulations $\Delta\Delta G_{calc}$ values tended towards unfavorability (i.e. more positive $\Delta\Delta G_{calc}$).
This is indicative of the fact that pREST is indeed improving sampling, but it is evident that our $\Delta\Delta G_{calc}$ are still far from convergence, given the RMSI is still large, especially for closed/open transformations.

\subsection*{Long simulations enhances protein conformational sampling from more exchanges}
Although we see improvements in sampling with pREST, the standard implemented time frame of 5ns clearly is not long enough to gain adequate sampling, particularly if we start from the protein closed state.
By running longer, we allow our simulations to perform more exchanges across replicas and thereby allow for better sampling of all conformational states at the relevant end state replicas.

Returning to our most extreme transformation, benzene to hexylbenzene, we have shown pREST alone does not facilitate adequate sampling of the open state (Fig~\ref{fig:c_opls3_rest1_1/RMSD-replica11}).
Now, when we run much longer we see far more sampling of the open protein conformational state in the final 10ns window (Fig~\ref{fig:c_opls3_rest1_1/45-55ns/RMSD-replica11}).
Note that, for closed-open transformations we use the final 15ns to compute the final free energies.
We only show the final 10ns in our RMSD/time analysis to avoid overcrowding data points.
In viewing all the replicas (Fig~\ref{fig:c_opls3_rest1_1/cmap-45-55ns}), we illustrate the dramatic increase in protein conformational sampling in stark contrast to our previous 5ns simulations (Fig~\ref{fig:c_opls3_rest1_1/colormap}).
An additional analysis comparing the protein-ligand contacts between the default and pREST protocol can be found in the Supporting Information (Fig. S1).

By simulating longer with pREST we dramatically increase our sampling of the intermediate and open protein states and almost entirely eliminate the dependence on the initial protein conformational state.
For the set of closed-open transformations the RMSI dramatically falls to +0.57 kcal/mol (Fig.~\ref{fig:C2O_xyplot_pREST},Table ~S\ref{tbl:C-O_pREST-40-55ns}) and a RMSI of +0.43 kcal/mol for the closed-intermediate (Fig.~\ref{fig:C2I_xyplot_pREST},Table ~S\ref{tbl:C-I_pRESText}).
Similarly, for experimental ligand transformations, our RMSE for protein closed simulations falls to +0.54 kcal/mol and the RMSI reduces to 0.31 kcal/mol (Fig.~\ref{fig:exp_xyplot_pREST},Table~S\ref{tbl:exp_pREST_set}).
Now, all our inconsistencies in the final calculated free energies and error from experiment fall within a much more reasonable range of less than +1 kcal/mol.  

It is very interesting to note that this overall level of agreement (a typical error of less than 1 kcal/mol) between our calculated values and experiment for the compounds for which we have affinity data -- the smaller compounds, for which the inconsistency is relatively low -- is actually quite good, better than what was reported in the previous work~\cite{FEPplus}.  
This may be due to the relative simplicity of the ligand modifications (methyl groups only) as well as the high quality of the experimental binding assay, ITC. 
In contrast, larger apparent errors in other tests may in some cases be due to lower quality experimental data. 
 
\section{Discussion}
In this study, we find that relative free energy calculations for this system can suffer from substantial convergence problems, likely due to the changes being related to the reorganization of a secondary structure element.
Although the protein conformational changes in T4 lysozyme (L99A) are extremely localized to a rearrangement of a single helix (Fig~\ref{fig:T4-L99A}), we still encounter challenges in sampling, likely due to the slow timescale associated with the reorganization of protein secondary structure.
These problems have clear implications for the accuracy of computed relative free energies in these cases. 
Particularly, we find that calculated relative free energies can depend on the initial protein conformational state by up to 4 kcal/mol.

By looking at alchemical transformations that involve a conformation change in the protein, we show the $\Delta\Delta Gs_{calc}$ may be sensitive to the initial protein conformational state when utilizing the default implemented FEP protocol.
This sensitivity occurs primarily when the alchemical transformation involves mutating ligands that mainly occupy the closed state (i.e. benzene to propyl) into ligands that occupy the intermediate state (i.e. (sec-)butyl) or, especially, the open state (pentyl/hexyl) (Table~\ref{tbl:expdata}).
By our RMSD analyses, we show the protein remains trapped in its initial state throughout the simulation when using the implemented default protocol.
Because it remains trapped, we are unable to adequately sample the necessary protein conformational states and thereby obtain inconsistent $\Delta\Delta Gs_{calc}$ depending on whether we start simulations from the open or closed protein configuration.  
This inconsistency can be large, especially for the larger compounds which bind primarily in the open conformation, for which we do not have experimental binding affinities. 
However, interestingly, when we start our simulations from the open structure, the lack of adequate sampling of the closed structure does not lead to particularly large errors for the smaller ligands which bind primarily in the closed structure (Table~S\ref{tbl:C-I}). 

Without prior knowledge of preferred protein conformational states on ligand binding, we can arrive at very different binding affinity predictions based on the initial protein state being used in the simulation.
By starting from the protein closed state and growing the ligand we obtain $\Delta\Delta G_{calc}$ values that appear overly positive or unfavorable due to high protein-ligand energy strain and inability to sample the open state, for this system.

However, when we begin with the protein open state our $\Delta\Delta G_{calc}$ values appear slightly too negative or favorable, resulting from inability to sample the closed state and not encountering protein-ligand strain. 
If we only had the crystal structure of the closed protein-ligand complexes, we would blindly conclude that the much larger, open-ligands bind to T4 lysozyme worse than smaller ones.
On the other hand, if only the open protein-ligand complexes were available, the opposite would be concluded in that larger ligands are better binders than smaller ligands; experimental data will be needed to determine which is the case. 

By including key residues into the REST region and simulating longer, we reduce the $\Delta\Delta G_{calc}$ dependence on the initial protein configuration to a more reasonable range of less than 1 kcal/mol.
Through expanding the REST region, intermediate lambda windows are able to more easily access the intermediate and open conformations by effectively heating key residues that facilitated protein motion, illustrated in Fig~\ref{fig:c_opls3_rest1_1/colormap}.
Further, by simulating longer we allow for more exchanges between replicas, which in turn enhances sampling at our physically relevant end state replica (Fig~\ref{fig:c_opls3_rest1_1/cmap-45-55ns}).   
With these modifications to the default protocol, we almost completely converge our $\Delta\Delta G_{calc}$ to the same value regardless of the starting protein conformation.

Generally, our brute-force approach of simulating longer and multiple trials with varied protein structures is not a desirable or feasible approach, especially in early drug discovery phases.
At the industrial level, ligand libraries can be large---driving computational cost exponentially if we simulate longer---or experimental structures can be sparse for new therapeutic protein targets.
For future studies, approaches using Markov State Models (MSMs)\cite{MSM} can potentially be of great use for identifying discrete protein conformations.
MSMs build a representation of the conformational space from batches of short molecular simulations, whereby the discrete states and transition rates between them can be determined in an efficient manner.
Utilizing MSMs can thereby provide useful insight on the various protein conformational states before running free energy predictions.

\section{Conclusions}
Overall, we have shown that the presence of kinetically distinct protein conformational states could impact the accuracy of free energy calculations by up to 2-5 kcal/mol, especially so when the R-group modification introduces a severe steric clash with the receptor.
It would have been especially challenging as there would essentially be no indicators that the final free energies were sensitive to the initial protein configuration.
Only from prior knowledge of the discrete states and by our tedious systematic trials were we able to identify and address the bias in our final calculated free energies.  
It should be comforting that even without this prior knowledge, good to very good agreement with experiment is obtained for the majority of ligands. 
Further, one should keep in mind that this study was carefully designed to probe the potential difficulties caused by these conformational changes; most lysozyme ligands studied previously do not induce such significant conformational changes. 

Although alchemical free energy calculations have shown tremendous recent successes on a variety of protein targets\cite{FEPplus}, we demonstrate challenges in protein sampling remain. 
Using T4 lysozyme (L99A) as our simple model system, we highlight sampling problems even from a relatively small (~1-3.5\AA) and localized single helix rearrangement in response to a series of growing ligands.
Through this study, we show using a typical 5ns simulation with only ligand atoms in the REST region, yields free energies that are sensitive to the initial protein conformation.
This is especially true for ligand perturbations that induce large protein conformational changes.
By longer simulation times and expansion of the REST region to include key protein residues, we were able to get converged predictions and nearly eliminate the depdendence on the starting conformation, even for some challenging cases.
This study demonstrates that special attention and care should be exercised when performing alchemical free energy calculations where regions of flexibility surround the binding site.
More importantly, prior to performing binding free energy calculations, we present strong evidence on the importance of identifying the occurrence of protein conformational changes upon ligand binding, 

\begin{acknowledgement}
   N.M.L. thanks Dmitry Lupyan and Joseph Goose for helpful discussions and technical support. Financial support for N.M.L. was provided by Schr\"{o}dinger and the National Science Foundation Graduate Research Fellowship (DGE-1321846). D.L.M. appreciates financial support from the National Institutes of Health (1R01GM108889-01). L.W. and R.A. are supported by Schr\"{o}dinger.
\end{acknowledgement}

\include{T4figs}

\begin{suppinfo}

\section*{Simulation Interactions Diagrams}
Using the `Simulation Interactions Diagram' tool provided with Desmond, we count the number of hydrophobic contacts between the ligand and each of the surrounding protein residues at each frame.
Hydrophobic contacts are categorized into three groups: $\pi$-Cation, $\pi-\pi$, and Other.
The $\pi$-Cation contact is defined as an aromatic and charged group that are within 4.5\AA.
$\pi-\pi$ contacts are when two aromatic groups are stacked face-to-face or face-to-edge.
Hydrophobic contacts that are classified as Other are when hydrophobic side-chains are within 3.6\AA of a ligand's aromatic or aliphatic carbons.

Through this protein-ligand contact analysis, we can see if the ligand forms the same interactions regardless of the initial protein configuration.
In Figure~\ref{fig:contactmap}, we analyze the hydrophobic contacts for hexylbenzene in either the protein closed (purple) or open (green) MD simulations and represent the difference in red.
Here, we represent the number of protein-ligand hydrophobic contacts as a percentage for the length of the MD simulations.
Again, we analyze frames from 0-5ns with the default protocol and 45-55ns with the pREST protocol.

Given hexylbenzene prefers the open conformation, we should expect to see Val111 contact the hexylbenzene at a percent closer to that of the protein open simulation.
Using the default protocol, we show Val111 is in contact with hexylbenzene for 40.4\% of the protein closed simulation and 9.6\% of the protein open simulation, approximately a 30.8\% difference in the interaction percentage.
On the other hand, with the pREST protocol---where Val111 is included into the REST region---this difference dramatically reduces to 0.5\%.
Now, regardless of our initial protein starting conformation, Val111 comes in contact with the ligand 7.9\% and 8.4\% of the protein closed and open simulations, close to what we would have expected judging from our protein open simulation using the default protocol.
Furthermore, we can compute the mean absolute difference (MAD) across all residues to yield 10.5\% for the default protocol and 2.8\% with the pREST protocol.
Overall, the reduction in the difference in protein-ligand contacts indicates the bias from the initial protein configuration is almost entirely eliminated from using pREST.

\setcounter{figure}{0} 
\begin{figure}[!ht]
\begin{subfigure}{\textwidth}
   \centering
   \frame{\includegraphics[scale=0.6]{Figures/ContactMap/benzene-hexylbenzene_CM_def.png}}
\end{subfigure}
\centering
\begin{subfigure}{\textwidth}
  \centering
  \frame{\includegraphics[scale=0.6]{Figures/ContactMap/benzene-hexylbenzene_CM_pREST.png}}
\end{subfigure}
\caption{Bar plot of interaction percentage of protein-ligand hydrophobic contacts over the course of a trajectory. 
From the benzene to hexylbenzene transformation, these plots correspond to the hexylbenzene ($\lambda_{11}$) state from closed (purple) and open (green) simulations using the default versus pREST protocols.
The mean absolute difference (MAD) in interaction percentage is represented in red.
MAD for the default protocol is 10.5\% and 2.8\% for the pREST protocol.
}
\label{fig:contactmap}
\end{figure}

\include{T4tables}

\section*{Data Sets and Analysis Tool Download}
All related data sets and analysis tools are available online and described below.
The files can be downloaded at http://n2t.net/ark:/b7280/d1js3b (doi:10.7280/D1JS3B). 
\begin{description}
\item [Prepared protein-ligand crystal structures:]\texttt{xtal\_prepped.tar.gz}\\
Contains the reference protein-ligand crystal structures that have been aligned and prepared through the `Protein Preparation Wizard' tool as \texttt{.mae} files.
\item [RMSD analysis:]\texttt{scripts.tar.gz}\\
Contains the scripts \texttt{RMSD-analysis.tcl} and \texttt{plot-RMSD.py} used for RMSD analysis. A \texttt{README.txt} file has been provided which details how to generate the RMSD/time and Colormap plots found in the paper. 
\item [FEP data:]\texttt{C-I\_default.tar.gz}, \texttt{C-I\_pPREST.tar.gz}, \texttt{C-O\_default.tar.gz}, \\ \texttt{C-O\_pREST.tar.gz}, \texttt{EXP\_default.tar.gz,} and \texttt{EXP\_pREST.tar.gz}\\
Contains archived data from the FEP calculations for each alchemical transformation set, where within each archived set are directories for individual FEP calculations. These contain input \texttt{.mae} files containing the protein/ligand structures, the simulation configuration \texttt{.msj} files, energy files, and a \texttt{README.txt} that summarizes information about each FEP calculation. Complete trajectories are included.
\end{description}



\end{suppinfo}

\clearpage
\bibliography{references}



\end{document}
